### PR TITLE
PRL-3592 - Download application form

### PR DESCRIPTION
**Pull request by hcarslaw**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/application-within-proceedings/download-form/content.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import { getCasePartyType } from '../../../steps/prl-cases/dashboard/utils';
+import { getApplicationDetails } from '../utils';
+
+export const en = {
+  title: 'Download the form',
+  explainRequest: 'You can explain your request by completing and uploading the form',
+  nextSteps: 'Next steps',
+  line1: 'Download the form',
+  from: 'from',
+  govUk: 'GOV.UK (opens in a new tab)',
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/download-form/content.ts:13]: Again make sure contents are in array and not split as lines

File: src/main/steps/application-within-proceedings/download-form/content.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  onlyContinue: 'Parhau',
+  cancel: 'Canslo',
+};
+
+const languages = {
+  en,
+  cy,
+};
+
+const downloadUrls = {
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/download-form/content.ts:44]: create a method which will return the downloadform url

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/application-within-proceedings/download-form/content.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/download-form/content.ts]: Changed to use array with whole lines

File: src/main/steps/application-within-proceedings/download-form/content.ts (hunk 2 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/download-form/content.ts]: Moved to config in utils file

File: src/main/steps/application-within-proceedings/download-form/content.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  warningText: 'You need to return to upload your application form for your request to be completed.',
+  onlyContinue: 'Continue',
+  cancel: 'Cancel',
+};
+
+export const cy: typeof en = {
+  title: 'Download the form (welsh)',
+  explainRequest: 'You can explain your request by completing and uploading the form (welsh)',
+  nextSteps: 'Next steps (welsh)',
+  bulletPoints: [
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/download-form/content.ts:27]: change it to listItems

File: src/main/steps/application-within-proceedings/utils.ts [hunk 4]
```diff
@@ -115,6 +117,8 @@ const applicationTypesConfiguration: AWPApplicationTypesConfig[] = [
       C100: '£167',
       FL401: '£167',
     },
+    applicationDownloadUrl:
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/utils.ts:120]: change it to applicationFormUrl

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/application-within-proceedings/utils.ts (hunk 4 above)
Comment (hcarslaw) [reply src/main/steps/application-within-proceedings/utils.ts]: Changed to applicationFormUrl

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**